types/react/index.d.ts

```diff
@@ -1921,6 +1921,17 @@ declare namespace React {
         wrap?: string;
     }
 
+    type HTMLAttributeReferrerPolicy =
+        | ''
```

Comment:
it's a bummer that this is in the spec.  any idea what `''` would be useful for (other than type-system/developer-experience ergonomics)